app/authentication/login/login.component.e2e-spec.js

```diff
@@ -0,0 +1,31 @@
+"use strict";
+var protractor_1 = require("protractor");
+describe('Login', function () {
+    beforeEach(function () {
+        protractor_1.browser.get('/login');
+    });
+    it('should have an input and submit button', function () {
+        var allInputElements = protractor_1.element.all(protractor_1.by.css('md-card md-card-content form input'));
+        var inputElement = protractor_1.element(protractor_1.by.css('md-card md-card-content form input'));
+        var submitButton = protractor_1.element(protractor_1.by.css('md-card md-card-content form button'));
+        var loginCardTitle = protractor_1.element(protractor_1.by.css('md-card md-card-title h3'));
+        protractor_1.browser.wait(function () {
+            return inputElement.isPresent();
+        }, 60000);
+        expect(inputElement.isPresent()).toEqual(true);
+        expect(submitButton.isPresent()).toEqual(true);
+        expect(protractor_1.element(protractor_1.by.css('md-card md-card-title h3')).getText()).toEqual('Geben Sie Ihre Daten ein, um sich einzuloggen');
```

Comment:
__[jshint]__
* 'expect' is not defined.

<!-- Comment by [SideCI](https://sideci.com) -->